pkg/sysinfo/sysinfo_linux.go

```diff
@@ -32,6 +33,11 @@ func New(quiet bool) *SysInfo {
 		sysInfo.AppArmor = true
 	}
 
+	// Check if Seccomp is supported.
+	if _, _, err := syscall.RawSyscall(syscall.SYS_PRCTL, syscall.PR_GET_SECCOMP, 0, 0); err != syscall.EINVAL {
```

Comment:
That won't work though because we can't check the container PID without running seccomp on it and having it fail :(